src/components/Header.js

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+	// 프로필 누구인지
+	const [person, setPerson] = useState(true);
+	
+	const inputTagStyle = {		// 메시지 입력창
+		display: 'flex',
+		height: '100%',
+		width: '70px',
+		borderRadius: '30%',
+	}
+	
+	const showWhoActiveTextStyle = {	// "OO 현재 활동중"
+		displya: 'flex',
+		paddingLeft: '15px',
+		alingContent: 'center',
+
+		paddingTop: '5px'	// 글자 가운데 정렬이 안되어서 임시방편
+	}
+
+	const activeTextStyle = {
+		display: 'flex',
+	}
+	
+	// ChattingScreen컴포넌트에 현재 프로필 전달
+	const formSubmit = e => {
+		e.preventDefault();
+		props.onSubmit(person);
+	}
+
+	// 프로필 클릭 시 변경
+	const onChangePerson = e => {
+		setPerson(!person);
+	}
+
+	return (
+		<Wrapper>
+			<form onSubmit={formSubmit}>
+				<input style={inputTagStyle} type="image" onClick={onChangePerson} 
```

Comment:
이건 그냥 `<button>` 태그로 하면 좋을 것 같아용

1. input 태그를 이미지로 지정
2. form으로 감싸기
3. onSubmit onChangePerson 2가지 이벤트를 전달
이들은 불필요하게 복잡하고 중복되는 코드인 것 같아요! form으로 감싸지 말고, img 태그에 onClick속성에 핸들러를 지정해주는 것이 좋아보여요

```suggestion
				<img src={...} onClick={handleProfileClick} 
```